variables.tf

```diff
@@ -72,19 +79,74 @@ variable "cloud_connection_speed" {
 #####################################################
 
 variable "tags" {
-  description = "List of Tag names for IBM Cloud PowerVS service"
+  description = "List of tag names for the IBM Cloud PowerVS service"
   type        = list(string)
   default     = null
 }
 
 variable "cloud_connection_gr" {
-  description = "Enable global routing for this cloud connection.Can be specified when creating new connection"
+  description = "Whether to enable global routing for this IBM Cloud connection. You can specify thia value when you create a connection."
   type        = bool
   default     = null
 }
 
 variable "cloud_connection_metered" {
-  description = "Enable metered for this cloud connection. Can be specified when creating new connection"
+  description = "Whether to enable metering for this IBM Cloud connection. You can specify thia value when you create a connection."
   type        = bool
   default     = null
 }
+
+variable "access_host_or_ip" {
+  description = "The public IP address for the jump or Bastion server. The address is used to reach the target or server_host IP address and to configure the DNS, NTP, NFS, and Squid proxy services."
+  type        = string
+}
+
+variable "squid_config" {
+  description = "Configuration for the Squid proxy to a DNS service that is not reachable directly from PowerVS"
+  type        = map(any)
+  default = {
+    "squid_enable"      = "false"
+    "server_host_or_ip" = "inet-svs"
+  }
+}
+
+variable "dns_forwarder_config" {
+  description = "Configuration for the DNS forwarder to a DNS service that is not reachable directly from PowerVS"
+  type        = map(any)
+  default = {
+    "dns_enable"        = "false"
+    "server_host_or_ip" = "inet-svs"
+    "dns_servers"       = "161.26.0.7; 161.26.0.8; 9.9.9.9;"
```

Comment:
Perhaps document where those dns server come from (eg: IBM Cloud doc for the 161 page) and rational for adding quad9 dns here (have we checked this is a preferred DNS provider as per corp policies?)

Reply:
Still not addressed